src/main/java/racingcar/domain/RaceRound.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.domain;
+
+import java.util.Collections;
+import java.util.List;
+
+public class RaceRound {
+    private static final int DEFAULT_MAX_DISTANCE = 0;
+    private final List<PlayerMove> playerMoveList;
+    private final MoveFactory moveFactory;
+
+    private RaceRound(final List<PlayerMove> playerMoveList, final MoveFactory moveFactory) {
+        this.playerMoveList = playerMoveList;
+        this.moveFactory = moveFactory;
+    }
+
+    public static RaceRound of(final List<PlayerMove> playerMoveList, final MoveFactory moveFactory) {
+        return new RaceRound(playerMoveList, moveFactory);
+    }
+
+    public void move() {
+        for (PlayerMove playerMove : playerMoveList) {
+            boolean isMove = moveFactory.isMove();
+            playerMove.move(isMove);
+        }
+    }
+
+    private int getMaxDistance() {
+        int presentMax = DEFAULT_MAX_DISTANCE;
+        for (PlayerMove playerMove : playerMoveList) {
+            presentMax = playerMove.getMaxDistance(presentMax);
+        }
+        return presentMax;
+    }
+
+    public void checkWinner() {
+        int maxDistance = getMaxDistance();
+        for (PlayerMove playerMove : playerMoveList) {
+            playerMove.checkWinner(maxDistance);
+        }
+    }
```

Comment:
PlayerMove를 List<>타입으로 갖고있다는 점을 활용해서 Stream API를 이용해볼 수도 있겠군요!

```suggestion
public void move() {
    playerMoveList.forEach(playerMove -> playerMove.move(moveFactory.isMove()));
}

private int getMaxDistance() {
    return playerMoveList.stream()
                         .mapToInt(PlayerMove::getDistance)
                         .max()
                         .orElse(DEFAULT_MAX_DISTANCE);
}

public void checkWinner() {
    int maxDistance = getMaxDistance();
    playerMoveList.stream()
                  .filter(playerMove -> playerMove.getDistance() == maxDistance)
                  .forEach(PlayerMove::checkWinner);
}
```